### ISSUE TITLE
.rpm does not verify: Payload SHA256 digest: BAD

### ISSUE THREAD

**Issue by akselbgs** [warpdotdev/Warp#4815]
### Describe the bug

The rpm package could not be installed on Fedora 

### To reproduce

1.Download the latest warp-terminal*rpm
2.  Navigate to the downloaded directory
3.
```shell
rpm -i warp-terminal*.rpm
``` 
or
```shell
dnf in warp-terminal*.rpm

``` 

### Expected behavior

The app to be installed sucesfully

### Screenshots

[rpm](https://app.warp.dev/block/JPeQ16DAMPPvgUQQix4t6s)
[dnf](https://app.warp.dev/block/j8f53T6QlGwsVqbIsl95h8)

### Operating system

Linux

### Operating system and version

Fedora 39

### Shell Version

 version 5.2.26(1)-release (x86_64-redhat-linux-gnu)

### Current Warp version

v0.2024.04.16.08.02.stable_03

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Additional context

System is up to date with latest fedora versions.
Kernel: 6.8.7-200.fc39.x86_64
Using warp appimage.

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by dannyneira** [comment]
Hi @akselbgs Sorry to hear about this. It's not clear from your shared blocks why the app wasn't able to be installed. Can you try installing it from another terminal, not the Warp app-image?

I tried to reproduce on fedora 39 VM and was successful from gnome-terminal. ![CleanShot 2024-04-24 at 16 40 12](https://github.com/warpdotdev/Warp/assets/16809145/ebb7e63a-0f63-4c20-b1e2-540791859aef)

**Comment by akselbgs** [comment]
Hi @dannyneira , thanks for the reply. I also tried both `gnome-terminal` and `gnome-software` but the result is same in all of them. I could not figure out the source of the issue either, the error message is not explanatory but always the same.
![Screenshot from 2024-04-25 15-07-08](https://github.com/warpdotdev/Warp/assets/59731920/cf0ddccf-5761-4f88-a5fe-d6065c43b6a4)

I also tried these commands with no luck;
`sudo dnf config-manager --set-disabled fedora-extras` (to make sure there are no conflicts )
`sudo update-crypto-policies --set LEGACY`
`sudo update-crypto-policies --set DEFAULT`
`sudo update-crypto-policies --set DEFAULT:SHA1`
